NebulaHost/PacketProcessors/Factory/Entity/CreatePrebuildsRequestProcessor.cs

```diff
@@ -91,8 +91,7 @@ public void ProcessPacket(CreatePrebuildsRequest packet, NebulaConnection conn)
                     if (canBuild)
                     {
                         FactoryManager.PacketAuthor = packet.AuthorId;
-                        // TODO: Fix this, as previewPose no longer exists
-                        //CheckAndFixConnections(btc, planet);
+                        CheckAndFixConnections(btc, planet);
```

Comment:
Can someone explain what behavior this method is supposed to fix? I believe I have updated it correctly but I don't know how to test

Reply:
Nevermind I see it is supposed to fix https://github.com/hubastard/nebula/pull/272 but I cannot reproduce the issue with the default simulated latency of the debug build